org.entangle_photo.Manager.json

```diff
@@ -0,0 +1,135 @@
+{
+    "app-id": "org.entangle_photo.Manager",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "entangle",
+    "rename-icon": "entangle",
+    "rename-desktop-file": "entangle.desktop",
+    "rename-appdata-file": "entangle.appdata.xml",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--metadata=X-DConf=migrate-path=/org/entangle-photo/manager/",
+        /* Needed for gvfs to work */
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        /* For USB, until there is a better way in flatpak */
+        "--device=all"
+    ],
+    "modules": [
```

Comment:
I've noticed some static libraries and headers lying around. You can add a global cleanup array, `["*.a", "*.la", "/lib/pkgconfig", "/lib/cmake", "/include"]` should be good enough.

Reply:
Oh yes, that's reasonabe.

Reply:
maybe it is time we have libgphoto2 in `shared-modules`... off the top of my head this is the 4th app that uses it. And I'm sure there are more.

Reply:
Yeah, in fact I feel that applies to all the deps I have there IMHO. Every single one of them is in use by at least one other application I know of that's based on GTK/GNOME.

Reply:
It's not a complicated to build dependency, we don't  want shared-modules to become a new dependency place thing. Apps are supposed to bundle the necessary libraries that aren't available in the SDK. Shared-modules is mostly for complicated, very common ones